samples/Humidity_BME280/.cproject

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
maybe IDE project files should not be included

Reply:
@slaff Now we have make `ide-eclipse` these could all be removed perhaps...

Reply:
> @slaff Now we have make ide-eclipse these could all be removed perhaps...

Yes, I agree. May be we can leave just the Sming Framework .[c]project files. Of course such a change should be a separate PR with update to the documentation too related to this change.